packages/docs/src/components/documentation.tsx

```diff
@@ -166,6 +169,26 @@ export class Documentation extends React.PureComponent<IDocumentationProps, IDoc
         Utils.safeInvoke(this.props.onComponentUpdate, activePageId);
     }
 
+    private getTitle() {
+        // Either get the title from the document, or return the default title
+        let title = "Blueprint - Documentation";
+        const docsTitle = document.getElementsByClassName("docs-title")[0].textContent;
+        // Check to make sure the textContent of docsTitle has a useable value
+        if (docsTitle !== undefined && docsTitle !== null) {
```

Comment:
`docsTitle != null` handles both of these cases in one fell swoop.

Reply:
I believe you mean, "swell foop."
classic javascript.